notebooks/05_Kubeflow_Pipeline/src/mnist/launcher/component.yaml

```diff
@@ -14,7 +14,6 @@ inputs:
 - {name: Evaluator Spec,            type: JSON,     default: '{}',      description: 'TFJob evaluator replicaSpecs.'}
 - {name: Tfjob Timeout Minutes,     type: Integer,  default: 1440,      description: 'Time in minutes to wait for the TFJob to complete.'}
 - {name: Delete Finished Tfjob,     type: Bool,     default: 'True' ,   description: 'Whether to delete the tfjob after it is finished.'}
-- {name: s3bucket,                 type: String,   default: 's3://eks-kubeflow-workshop-data/iris-example/tb-logs',         description: 'The S3 bucket name'}
```

Comment:
Because I found it's not necessary, we have already pass the argument of `s3bucket` into our `chief` and `worker` spec.

![image](https://user-images.githubusercontent.com/23116624/79013769-45e39700-7b1e-11ea-841f-06f9f02d5c74.png)